packages/data/src/redux-store/metadata/selectors.js

```diff
@@ -42,7 +67,9 @@ export function getIsResolving( state, selectorName, args ) {
  * @return {boolean} Whether resolution has been triggered.
  */
 export function hasStartedResolution( state, selectorName, args ) {
-	return getIsResolving( state, selectorName, args ) !== undefined;
+	return (
+		getResolutionState( state, selectorName, args )?.status !== undefined
```

Comment:
Since that `status` will always be defined if the state is defined, I guess we can just do this:

```suggestion
		getResolutionState( state, selectorName, args ) !== undefined
```

Reply:
This hinges on https://github.com/WordPress/gutenberg/pull/38669#discussion_r817348750, I removed the '?' for now